modules/geo-layers/src/mvt-layer/clip-extension.js

```diff
@@ -81,8 +81,9 @@ varying vec2 clip_commonPosition;
 };
 
 export default class ClipExtension extends LayerExtension {
-  getShaders(opts) {
+  getShaders() {
     const hasAnchor = 'instancePositions' in this.getAttributeManager().attributes;
```

Comment:
Nit: This checking to decided VS for FS clipping seems error prone, may be add a prop to each layer , similar to `shadowEnabled` ??

Reply:
This extension is supposed to work automatically without the user marking any layers, and I believe this is a reasonable default. I will add a prop that allows users to override the default behavior.